pkg/controller/quayecosystem/quayecosystem_controller.go

```diff
@@ -177,11 +183,12 @@ func (r *ReconcileQuayEcosystem) Reconcile(request reconcile.Request) (reconcile
 			return r.manageError(quayConfiguration.QuayEcosystem, redhatcopv1alpha1.QuayEcosystemQuaySetupFailure, err)
 		}
 
-		err = r.quaySetupManager.SetupQuay(quaySetupInstance)
-
-		if err != nil {
-			r.reconcilerBase.GetRecorder().Event(quayConfiguration.QuayEcosystem, "Warning", "Failed to Setup Quay", err.Error())
-			return r.manageError(quayConfiguration.QuayEcosystem, redhatcopv1alpha1.QuayEcosystemQuaySetupFailure, err)
+		if !quayEcosystem.Spec.SkipValidation {
```

Comment:
Not sure if this make sense... it allows the unit test to validate the deployment of clair, but if the skipValidation flag is being used then it makes the pods that are being setup in setupQuay() dead in the water. I'd like to try to avoid flags used strictly for testing if possible. Thoughts @sabre1041 ? I'm thinking we just won't be able to test much of the reconcile cycle within the fake client and instead test it mostly within the E2E tests.

Reply:
I would not recommend the `skipValidation` approach. Surely we should be able to use the fake client and related testing tools to implement this

Reply:
Removed skipValidation